paper-checkbox.html

```diff
@@ -215,13 +222,19 @@
       }
     </style>
 
+    <template is="dom-if" if="[[leftSideLabel]]">
+      <div id="checkboxLabel" class="checkboxLabel-left"><content></content></div>
+    </template>
+
     <div id="checkboxContainer">
       <div id="checkbox" class$="[[_computeCheckboxClass(checked, invalid)]]">
         <div id="checkmark" class$="[[_computeCheckmarkClass(checked)]]"></div>
       </div>
     </div>
 
-    <div id="checkboxLabel"><content></content></div>
+    <template is="dom-if" if="[[!leftSideLabel]]">
+      <div id="checkboxLabel" class="checkboxLabel-right"><content></content></div>
+    </template>
```

Comment:
I think it might be cleaner to give the host `display: inline-flex;` and add
```
:host([left-side-label]) {
  flex-direction: row-reverse;
}
```
The padding CSS would need to change slightly too, but this way we wouldn't need any `dom-if`s. Thoughts?